fastestimator/network.py

```diff
@@ -75,7 +77,10 @@ def __init__(
         self.epoch_postprocessing = []
         self.epoch_models = set()
         self.epoch_state = dict()
-        self.scaler = None
+        self.mixed_precision = any([model.mixed_precision for model in self.models])
+
+        if self.mixed_precision and not all([model.mixed_precision for model in self.models]):
+            raise ValueError("All models need to be either mixed-precision or not")
```

Comment:
"Cannot mix full precision and mixed-precision models"

Reply:
fixed